contracts/minting/MintController.sol

```diff
@@ -89,14 +89,15 @@ contract MintController is Controller {
      * transaction to a minter and not worry about it being used to undo a removeMinter()
      * transaction.
      */
-     function incrementMinterAllowance(uint256 allowanceIncrement) onlyController public returns (bool) {
+     function incrementMinterAllowance(uint256 _allowanceIncrement) onlyController public returns (bool) {
+        require(_allowanceIncrement > 0, "Increment allowance must be greater than 0");
```

Comment:
Please change the error message to "Allowance increment must be greater than 0."  The phrase `Increment allowance` is a verb (which cannot be compared to 0) while `Allowance increment` is a noun.